clar2wasm/tests/standard/unit_tests.rs

```diff
@@ -52,7 +52,7 @@ fn test_add_uint() {
     // 1 + 0xffff_ffff_ffff_ffff_ffff_ffff_ffff_ffff = Overflow
     add.call(
         &mut store,
-        &[Val::I64(-1), Val::I64(-1), Val::I64(0), Val::I64(1)],
+        &[Val::I64(-1), Val::I64(-1), Val::I64(1), Val::I64(0)],
```

Comment:
This is not related to your current changes, but these should be switched. It's currently the same as the previous test.
```suggestion
        &[Val::I64(1), Val::I64(0), Val::I64(-1), Val::I64(-1)],
```